mcs/packages/Makefile

```diff
@@ -53,6 +53,14 @@ install-local:
 
 	(cd $(MSBUILD_ROSLYN_DIR); for asm in $(ROSLYN_FILES_TO_LINK_FOR_MSBUILD); do ln -fs ../../../../$(FRAMEWORK_VERSION)/$$asm . ; done)
 
+test-local: test-csi
+
+test-csi:
+	@echo 'Console.WriteLine ("hello world: " + DateTime.Now)' > csi-test.csx
```

Comment:
I think we should just add the file instead of generating it

Reply:
Ok. Are you happy with the place where I added this test? Not sure if `mcs/packages` is really the best place...

Reply:
The location is good